test/features/parsoid/transform.js

```diff
@@ -15,197 +15,167 @@ describe('transform api', function() {
     this.timeout(20000);
     let contentTypes;
     const server = new Server();
+    const client = new REST('');
     before(() => {
         return server.start()
         .then(() => {
             contentTypes = server.config.conf.test.content_types;
-            return preq.get({
-                uri: `${server.config.bucketURL('en.wikipedia.beta.wmflabs.org')}/html/${testPage.title}/${testPage.revision}`
-            });
+            return client.get(
+                `${server.config.bucketURL('en.wikipedia.beta.wmflabs.org')}/html/${testPage.title}/${testPage.revision}`
+            );
         })
         .then((res) => {
-            testPage.html = res.body;
+            testPage.html = res.text;
         });
     });
     after(() => server.stop());
 
     it('wt2html', () => {
-        return preq.post({
-            uri: `${server.config.baseURL('en.wikipedia.beta.wmflabs.org')}/transform/wikitext/to/html/${testPage.title}`,
-            body: {
-                wikitext: '== Heading =='
-            }
-        })
+        return client.post(
+            `${server.config.baseURL('en.wikipedia.beta.wmflabs.org')}/transform/wikitext/to/html/${testPage.title}`,
+            { wikitext: '== Heading ==' }
+        )
         .then((res) => {
             assert.deepEqual(res.status, 200);
-            assert.contentType(res, contentTypes.html);
+            assert.contentTypes(res, mwUtil.constructRegex([contentTypes.html]));
```

Comment:
I published a new version that makes contentType singular here. Can you update the package to 1.4.1 and update this assert accordingly?

Reply:
Thanks @clarakosi, `api-testing` has been updated from 1.4.0 to 1.4.1 and I was to use the singular `contentType` but @Pchelolo [advised](https://github.com/wikimedia/restbase/pull/1281#discussion_r518781424) that we replace all pattern matchings with `assert.match`.